### ISSUE TITLE
Why no caption data in the SpeechSynthesizer.Speak directive

### ISSUE THREAD

**Issue by mwmy2009** [alexa/avs-device-sdk#1699]
**IMPORTANT**: Before you create an issue, please take a look at our [Issue Reporting Guide](https://github.com/alexa/avs-device-sdk/wiki/Issue-Reporting-Guide).

## Briefly summarize your issue:
no caption data is there in the Json of SpeechSynthesizer.Speak directive when we use OAuth token with our own product. I can see captions if we use sampleApp with CBL. Just replace the CBL with our own authdelegate, caption data isn't in the json any more. What could be the problem? Other capbilities works fine.

## What is the expected behavior?
Caption data can be seen.

## What behavior are you observing?
I see no caption filed in the json of SpeechSynthesizer.Speak. Then no caption is presented.

## Provide the steps to reproduce the issue, if applicable:
Use our own authdelegate to get the access token, change the client id/product id in DeviceInfo.cpp

### What version of the AVS Device SDK are you using?
      1.18.0

### Tell us what hardware you're using:
- [ ] Desktop / Laptop
- [ ] Raspberry Pi
- [ ] Other - tell us more:
Android phone

### Tell us about your OS (Type & version):
- [ ] Linux
- [ ] MacOS 
- [ ] Raspbian Stretch
- [ ] Raspbian Jessy
- [ ] Other - tell us more:

**Comment by mvelegon-amzn** [comment]
Hi @mwmy2009 Thanks for reaching out. In order for us to investigate further could you please share DEBUG9 logs for with CBL auth delegate and with your own authdelegate for the same utterance?

**Comment by mwmy2009** [comment]
Hello
[CBL_what's_the_weather_in_NewYork.txt](https://github.com/alexa/avs-device-sdk/files/4598550/CBL_what.s_the_weather_in_NewYork.txt)
[ownauth_what's_the_weather_in_newyork.txt](https://github.com/alexa/avs-device-sdk/files/4598551/ownauth_what.s_the_weather_in_newyork.txt)

Thanks a lot for the quick feedback.

I attached two txt files (CBL vs. own authdelegate) with "what's the weather in Newyork".

**Comment by Kyle-Falconer** [comment]
@mwmy2009  I looked at the logs and saw that in `ownauth_what's_the_weather_in_newyork.txt`, it shows that captions are not present in the payload:

> `2020-05-08 18:47:51.525 5788-6131/wearable.a70mai.com.alexaexample I/Native: 2020-05-08 10:47:51.525 [ 17] 3 SpeechSynthesizer:captionsNotParsed:reason=keyNotFoundInPayload`

There are only two reasons for captions to not be present in the payload:
1. SpeechSynthesizer version < 1.3 or AudioPlayer < 1.4
2. The skill does not support captions, which only applies to AudioPlayer skills, since captions need to be explicitly added to AudioPlayer skills.

I also saw in the logs that it appears SpeechSynthesizer is set to 1.3. I would advise you verify that this version number is being sent correctly. Because captions are being sent otherwise, this is the only reason I can see for a SpeechSynthesizer response to not contain captions.
